src/commons/sagas/BackendSaga.ts

```diff
@@ -685,7 +685,8 @@ function* BackendSaga(): SagaIterator {
         gradingQuestion.grade = {
           xpAdjustment,
           xp: gradingQuestion.grade.xp,
-          comments
+          comments,
+          grader: { id: 0, name: '' }
```

Comment:
Yes this is for `checkIsNewQuestion`. I couldn't think of a better way to do it since all the values are reupdated in the Props whenever the save button is clicked. Would it be better if a new variable like `hasBeenGraded` is added instead?

Reply:
Could we make use of [`gradedAt`](https://github.com/source-academy/frontend/blob/master/src/features/grading/GradingTypes.ts#L86-L95) instead of adding a dummy grader here? 

If there are reasons why we can't do this, then yes, I think adding a new variable such as `hasBeenGraded` (if necessary) would better communicate what we are trying to do, than to add a dummy grader object.

Reply:
I changed it so it uses gradedAt instead of a dummy grader. Is my implementation ok?

Reply:
Looks better to me!